Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/actions/FilterActionOnClearFilter.java

```diff
@@ -40,7 +40,8 @@ public boolean execute() {
             analyst.clearFilter();
             return true;
         } catch (Exception e) {
-            Messagebox.show("Error in clearing filter. Error message: " + e.getMessage());
+            // LOGGER.error("Error in clearing filter. Error message: " + e.getMessage());

```

Comment:
Can this be removed, or better, uncommented? If uncommented, include `e` in the logging call.